akka-cluster-sharding-typed/src/test/scala/akka/cluster/sharding/typed/ReplicatedShardingSpec.scala

```diff
@@ -91,7 +92,7 @@ object ReplicatedShardingSpec {
                 replyTo ! Texts(state)
                 Effect.none
             },
-          (state, event) => state + event)
+          (state: Set[String], event: String) => state + event)
```

Comment:
why the additional types in several places in this class?

Reply:
i was debugging class cast issues but that was caused by replicated events being sent to the wrong proxies, will revert